examples/EL-attention/EL_attention_Demo.ipynb

```diff
@@ -0,0 +1,694 @@
+            "Mohammad Javad Zarif is the Iranian foreign minister. He has been John Kerry's opposite number in securing a breakthrough in nuclear talks. He received a hero's welcome as he arrived in Iran on a sunny Friday morning. But there are some facts about Zarif that are less well-known.\n",
+            "Bob Barker returned to \"The Price Is Right\" for the first time in eight years. The 91-year-old hosted the show for 35 years before stepping down in 2007. He handled the first price-guessing game, the classic \"Lucky Seven,\" before turning hosting duties over to Drew Carey.\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "nmfO-Pj-7sYa"
+      },
+      "source": [
+        "!cp 320_test.hypo test.hypo\n",
+        "!cp data/test.target test.target"
+      ],
+      "execution_count": 12,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "etI5Qu4o6tWm"
+      },
+      "source": [
+        "## Evaluate"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fsPa-OjU7ASr"
+      },
+      "source": [
+        "Follow https://github.com/pytorch/fairseq/tree/master/examples/bart to install *files2rouge*"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "D4YkacBQyyke",
+        "outputId": "81b3db13-85e7-4d7b-9aaa-54fbd09e2ded"
+      },
+      "source": [
+        "!export CLASSPATH=/path/to/stanford-corenlp-full-2016-10-31/stanford-corenlp-3.7.0.jar\n",
+        "\n",
+        "# Tokenize hypothesis and target files.\n",
+        "!cat test.hypo | java edu.stanford.nlp.process.PTBTokenizer -ioFileList -preserveLines > test.hypo.tokenized\n",
+        "!cat test.target | java edu.stanford.nlp.process.PTBTokenizer -ioFileList -preserveLines > test.hypo.target\n",
+        "!files2rouge test.hypo.tokenized test.hypo.target\n",
+        "# Expected output: (ROUGE-2 Average_F: 0.21227)"
+      ],
+      "execution_count": 13,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Error: Could not find or load main class edu.stanford.nlp.process.PTBTokenizer\n",
+            "Caused by: java.lang.ClassNotFoundException: edu.stanford.nlp.process.PTBTokenizer\n",
+            "Error: Could not find or load main class edu.stanford.nlp.process.PTBTokenizer\n",
+            "Caused by: java.lang.ClassNotFoundException: edu.stanford.nlp.process.PTBTokenizer\n",
+            "/bin/bash: files2rouge: command not found\n"
```

Comment:
Is it expected to show error messages here?

Reply:
This part cannot run. I will change this cell from code to text.